doc/manual/manual.tex

```diff
@@ -11169,7 +11169,7 @@ \subsection{How to write a plugin}
   \end{itemize}
   You can then just run the commands
     \begin{verbatim}
- cmake -DAspect_DIR=/path/to/aspect .
+ cmake -DAspect_DIR=/path/to/aspect/build/ .
```

Comment:
Above you have omitted the trailing slash.

Reply:
good point, I prefer it with a slash. I hope this is acceptable.

Reply:
I've got no preference. This is ok with me.